nntrainer/layers/common_properties.h

```diff
@@ -69,6 +104,84 @@ class Unit : public nntrainer::Property<unsigned int> {
   bool isValid(const unsigned int &v) const override { return v > 0; }
 };
 
+/**
+ * @brief TensorsSpec property, this defines tensors shape of output
```

Comment:
this defines tensors shape~ of output~? Then reuse this can be reused input_shape as well.

Reply:
right. it can be used for input_shape as well.